### PR TITLE
Close Issues #26 Organize code of module graphManager.js, #27 Organize code of module navBarManager.js

### LINKED ISSUE #26: [JavaScript Improvements] - Organize code of module graphManager.js

### LINKED ISSUE #27: [JavaScript Improvements] - Organize code of module navBarManager.js

### PR DESCRIPTION
**Pull request by CjuanRcris7**
### Description
This PR add the module graph.js to contain the code related to graph visualization.
This PR also add the module navBar.js to contain the code related to sidebar.

### Issues
Closes #26 #27

### CODE REVIEW

File: FRONTEND/src/modules.js/navBar.js [hunk 1]
```diff
@@ -0,0 +1,354 @@
...
+        },
+    });
+};
+
+
+
+// ------------------------------ Function-7 ------------------------------
+// Exportamos la función de inicialización del autocompletado
+function initAutocomplete(toolTopicData, addNodesFn, toolImage, databaseImage, topicImage) {
+    $(function () {
```
Comment (JessicaFM) [inline FRONTEND/src/modules.js/navBar.js]: Do you really need this jquery init here again?

**Review by JessicaFM — CHANGES_REQUESTED** [review]
Some changes

File: FRONTEND/src/modules.js/navBar.js (hunk 1 above)
Comment (CjuanRcris7) [reply FRONTEND/src/modules.js/navBar.js]: PR Request solved. Eliminate unnecessary jquery of function initAutocomplete of FRONTEND/src/modules.js/navBar.js

File: FRONTEND/src/modules.js/navBar.js [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+        "</div></div></li>"
+      ).appendTo(ul);
+    }
+  };
+}
+
+
+
+// ------------------------------ Function-8 ------------------------------
+function removeLegend() {
```
Comment (JessicaFM) [inline FRONTEND/src/modules.js/navBar.js:255]: do we need a const for this?
And need to be jQuery prob.

**Review by JessicaFM — CHANGES_REQUESTED** [review]
Some little update

File: FRONTEND/src/modules.js/navBar.js (hunk 2 above)
Comment (CjuanRcris7) [reply FRONTEND/src/modules.js/navBar.js:255]: Fixed!
const removed and use JQuery to select HTML element.

**Review by JessicaFM — APPROVED** [review]
OK

**Pull request merged**